virttest/utils_kernel_module.py

```diff
@@ -0,0 +1,111 @@
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+"""Handle Linux kernel modules"""
+
+
+class KernelModuleHandler(object):
+    """Class handling Linux kernel modules"""
+
+    def __init__(self, module_name):
+        self._module_name = module_name
+        self._was_loaded = True
+        self._config_backup = ""
+        self._backup_config()
+
+    def load_module(self, force=True, params=""):
+        """
+        Load module with given parameters
+
+        :param force: Force load if currently not loaded. Default is True as restore undoes loading.
+        :param params: Parameters to load with, 'key1=param1 key2=param2 ...'
+        """
+
+        if not force and not self._was_loaded:
+            logging.info("Module %s isn't loaded. Set force=True to force loading." % self._module_name)
```

Comment:
Good, but it'd be really neat to check values of `params` and only warn in case they differ. What do you think?